From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Subject: mm: no need to export mm_kobj
Date: Fri, 4 Aug 2023 08:43:37 +0200

There are no modules using mm_kobj, so do not export it.

Link: https://lkml.kernel.org/r/2023080436-algebra-cabana-417d@gregkh
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Reviewed-by: Mike Rapoport (IBM) <rppt@kernel.org>
Reviewed-by: Miaohe Lin <linmiaohe@huawei.com>
Reviewed-by: David Hildenbrand <david@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/mm_init.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/mm_init.c~mm-no-need-to-export-mm_kobj
+++ a/mm/mm_init.c
@@ -154,7 +154,6 @@ early_param("mminit_loglevel", set_mmini
 #endif /* CONFIG_DEBUG_MEMORY_INIT */
 
 struct kobject *mm_kobj;
-EXPORT_SYMBOL_GPL(mm_kobj);
 
 #ifdef CONFIG_SMP
 s32 vm_committed_as_batch = 32;
_
